events/api.py

```diff
@@ -1751,17 +1759,73 @@ def _create_or_update_registration_user_accesses(
                 else:
                     raise
 
+        return user_accesses
```

Comment:
This added return value is not used anywhere.

Reply:
Return value removed